concordium-node/README.md

```diff
@@ -10,6 +10,7 @@
   v22.12.06 is what we currently use. Either build from the v22.12.06 tag of the repository using CMake and copy the `flatc` binary somewhere in your PATH, or download a released binary from <https://github.com/google/flatbuffers/releases/tag/v22.12.06> and place it somewhere in your PATH.
 - protobuf >= 3.15
 - LLVM and Clang >= 3.9
+- As noted in the [caveats](#caveats) section below, you need to build the [Concordium Consensus](../concordium-consensus/) project before building this project. So you also need the [dependencies listed in the README for Concordium Consensus](../concordium-consensus#build-requirements).
```

Comment:
```suggestion
- As noted in the [caveats](#caveats) section below, you need to build the [Concordium Consensus](../concordium-consensus/) project before building this project. 
  So you also need the [dependencies listed in the README for Concordium Consensus](../concordium-consensus#build-requirements).
```

Reply:
The link to the section `#build-requirements` also does not seem to work. It just links to the page, not to the section.

Reply:
Hmm, not sure why that doesn't work.
If I look at the generated page on GitHub and _copy/paste_ the link, then the `#build-requirements` part is included and works. But if I just click the link, then it doesn't work.

Reply:
I'm guessing if you included a full link, not a relative one, it would work.

Reply:
Yep, that works.